tests/test_convert_lit_checkpoint.py

```diff
@@ -101,3 +101,47 @@ def test_against_original_gpt_neox():
     ours_y = ours_model(x)
     theirs_y = theirs_model(x)["logits"]
     torch.testing.assert_close(ours_y, theirs_y)
+
+
+def test_maybe_unwrap_statedict(tmp_path):
```

Comment:
```suggestion
def test_maybe_unwrap_state_dict(tmp_path):
```

Reply:
implemented in [169acf6](https://github.com/Lightning-AI/lit-gpt/pull/341/commits/169acf6aed0f50e4f6c929274a7902c46bb4815f)